Src/EmotivUnityItf.cs

```diff
@@ -1150,18 +1151,28 @@ private string Md5(string s)
 
         private void InitForAuthentication(string clientId, string clientSecret)
         {
+            string server = "cerebrum.emotivcloud.com";
```

Comment:
we have build define `DEVELOPMENT_BUILD ` and `PRODUCTION_BUILD `. 
should config different url for different environment .